blazor-toc.html

```diff
@@ -283,6 +283,9 @@
 					<li>
 						<a href="/blazor/common/how-to/use-blazor-report-viewer">Syncfusion ReportViewer in Blazor application</a>
 					</li>
+					<li>
+						<a href="/blazor/common/how-to/port-blazor-server-maui">Port Blazor Server Side App to MAUI Blazor Hybrid App</a>
```

Comment:
Port Blazor Server App to MAUI Blazor Hybrid App

Reply:
Updated as Blazor Server App